qiita_pet/handlers/download.py

```diff
@@ -222,3 +230,59 @@ def get(self, study_id):
         self.set_header('Content-Disposition',
                         'attachment; filename=%s' % zip_fn)
         self.finish()
+
+
```

Comment:
These two handlers are doing pretty much the same, can this be factored in to a base class or in a helper function?